### ISSUE TITLE
notify members when their messages are deleted

### ISSUE THREAD

**Issue by Little-Green-Guy** [CyberDiscovery/meta#301]
# notify members when their messages are deleted

### *A description of the issue:*
currently when our messages are deleted we may not realise or have no idea why, possibly not know that the message is unreasonable in the channel/server, and may continue to post similar things which then need to be deleted by mods.

### *Proposed Solution:*
dm members when their message is deleted and a (brief) reason why.
If mods are prepared to discuss the reason with the member, and they later decide that actually the message was ok, and it was due to the need to act quickly that the message was deleted, it is then more likely that this will be communicated and the user knows that that type of message is ok, and can send it again if they care enough.

### *Additional Comments:*
Although it may be inconvenient to have to explain reasons for deleting messages for mods, I often see that deleted messages are often protested when noticed, and mods have to explain why anyway.
Purges will also have to be handled differently. I think this may be less of an issue as the reason is often more obvious.

**Comment by lightspeedana** [comment]
This is an interesting idea - we could definitely communicate this better. Nonetheless, explaining every deletion can be tedious - I don't think any of us would mind doing so if asked?

**Comment by Little-Green-Guy** [comment]
i dont know how many deletions you do but sometimes we just dont know something was deleted or dont know why and it can be frustrating.

**Comment by NicholasG04** [comment]
lmao it's just a deleted message but like a little thing in chat would probz be helpful as to a short explanation

**Comment by lightspeedana** [comment]
I think we'd be happy providing those short explanations in chat if asked for, but for every message ever deleted it would probably be excessive. Otherwise, perfectly happy to explain anytime needed if asked for in specific cases 😄

**Comment by lightspeedana** [comment]
Sometimes it's almost common sense, but other times I can understand the frustration. So I think it can be settled as we'd provide explanations when asked for, or if the situation is a bit hard to follow and warrants it.

**Comment by Little-Green-Guy** [comment]
first time a meta issue has been responded to and resolved (albeit rejected) in a day? I think thats a first lol.

**Comment by lightspeedana** [comment]
Sorry, that's my bad 😅 although, speed is something I am trying to work on. Hopefully can get more done sooner in future!

**Comment by Little-Green-Guy** [comment]
no there's nothing bad about it!
